libpcsxcore/psxinterpreter.c

```diff
@@ -509,6 +530,7 @@ void psxDIV() {
 		_i32(_rLo_) = _i32(_rRs_) >= 0 ? 0xffffffff : 1;
 		_i32(_rHi_) = _i32(_rRs_);
 	}
+#endif
```

Comment:
You don't have to keep the whole thing as what it's doing is completely identical except the `} else if (_i32(_rRs_) == 0x80000000 && _i32(_rRt_) == 0xFFFFFFFF)` block, so choose one of the blocks and only ifdef the `... _i32(_rRt_) == 0xFFFFFFFF` please.